### DISCUSSION TITLE
Vite dev server advice (many problems since migrating from webpack)

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#14331] [General - Components / Directives / etc]
I migrated from webpack to vite a while back, which was far from a painless process, but I reasoned that I would get so many benefits from the switch that it would be well worth it. Sadly, since migrating I find the dev environment to be much less stable and reliable than webpack with the following issues:
- depending on the file being changed, a hot reload can take forever compared to webpack and seems to rerun the same code over and over until it finally shakes out
- sometimes changes simply aren't recognized at all and necessitate a force reload in the browser
- if I leave the dev server running a few hours on my laptop and return, it will no longer work and I will need to restart it.
- Startup to usable preview is only about 25% faster than webpack

So at this point I am wondering: is it just me? Are there other things to check or try? I can't believe this is the experience others are having with vite.

**Comment by metalsadman** [comment]
I switched back to webpack once I've finished upgrading a very large project to vite, I didn't find it any faster and finding/configuring webpack equivalent plugins was quite a challenge in my brief experience. I would pick it up tho if I would start a new one to really dive into the hype.
